homeassistant/components/recorder/migration.py

```diff
@@ -120,6 +120,12 @@
     from . import Recorder
 
 LIVE_MIGRATION_MIN_SCHEMA_VERSION = 0
+
+MIGRATION_NOTE_OFFLINE = (
+    "Note: this may take several hours on large databases and slow computers. "
+    "Home Assistant will not start until the upgrade is completed. Please be patient!"
```

Comment:
```suggestion
    "Home Assistant will not start until the migration is completed. Please be patient!"
```
Should we also tell them that they should not reboot?

Reply:
It was intentional to use the word "upgrade", I think it's easier to understand than "migration"?

We could maybe say "do not turn off"?

Reply:
Ah that's fine! 

Yea let's add the message

Reply:
OK, how do you suggest to word it? "Please be patient and do not turn off or restart Home Assistant while the upgrade is in progress!"?